znai-reactjs/src/doc-elements/latex/InlinedLatex.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 TWO SIGMA OPEN SOURCE, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
it is a bit out of scope of the ticket. But if you can, could you start adding new components as `.tsx` ones? There is only one at the moment. But more will be coming soon. Being long on my plate.

Let me know if you want to do it as a separate PR.

Reply:
Done, happy to do so moving forward (and it's preferred!).